prac_06/guitar.py

```diff
@@ -0,0 +1,16 @@
+class Guitar:
+    def __init__(self, name: str, year: int, cost: float):
+
+        self.name = name
+        self.year = year
+        self.cost = cost
+
+    def __str__(self) -> str:
```

Comment:
same with the get_age function, I'm unsure why you have the "-> str". This is unnecessary